File: PrivateVault.ai-main/sdk/python/privatevault_sdk/evidence.py
from privatevault_sdk.client import Client


def export_evidence(
    client: Client,
    tenant_id: str,
    start: str,
    end: str,
    bundle_name: str | None = None,
    idempotency_key: str | None = None,
):
    body = {"tenant_id": tenant_id, "start": start, "end": end}
    if bundle_name:
        body["bundle_name"] = bundle_name
    resp = client.request(
        "POST",
        "/evidence/export",
        json_body=body,
        idempotency_key=idempotency_key,
    )
    resp.raise_for_status()
    return resp.json()

File: PrivateVault.ai-main/sdk/node/src/evidence.js
export async function exportEvidence(client, { tenantId, start, end, bundleName, idempotencyKey }) {
  const body = { tenant_id: tenantId, start, end };
  if (bundleName) {
    body.bundle_name = bundleName;
  }
  const resp = await client.request('POST', '/evidence/export', {
    jsonBody: body,
    idempotencyKey,
  });
  if (!resp.ok) {
    throw new Error(`evidence export failed: ${resp.status}`);
  }
  return resp.json();
}
